packages/clay-core/src/table/Column.tsx

```diff
@@ -56,29 +74,86 @@ export const Column = React.forwardRef<HTMLTableCellElement, Props>(
 			className,
 			delimiter,
 			expanded,
+			keyValue,
+			sorting,
 			textAlign,
+			textValue,
 			truncate,
 			wrap = true,
 			...otherProps
 		},
 		ref
 	) {
+		const {onSortChange, sort, sortDescriptionId} = useTable();
 		const scope = useScope();
 		const As = scope === Scope.Head ? 'th' : 'td';
 
 		return (
 			<As
 				{...otherProps}
+				aria-describedby={
+					scope === Scope.Head && sorting
+						? sortDescriptionId
+						: undefined
+				}
+				aria-sort={
+					scope === Scope.Head && sorting
+						? sort && keyValue === sort.column
+							? sort.direction
+							: 'none'
+						: undefined
+				}
 				className={classNames(className, {
 					'table-cell-expand': truncate || expanded,
 					[`table-cell-${delimiter}`]: delimiter,
 					[`table-column-text-${textAlign}`]: textAlign,
 					[`text-${align}`]: align,
 					'table-cell-ws-nowrap': !wrap,
+					'table-head-title': scope === Scope.Head,
 				})}
 				ref={ref}
 			>
-				{truncate ? (
+				{scope === Scope.Head && sorting ? (
+					<a
```

Comment:
Why is this an anchor instead of a button?

Reply:
It seems that `button` doesn't have the correct styling inside the header, I need to add this to the CSS but I haven't gone ahead with it yet because I will remove the link in the next interactions and keep the action in `th` instead.